scoper.inc.php

```diff
@@ -33,58 +33,10 @@
 
 declare(strict_types=1);
 
-use Isolated\Symfony\Component\Finder\Finder;
-
-// see https://github.com/humbug/php-scoper/blob/0ed64857e86ae9ea4f5889c34e153f2cdbe968b5/docs/further-reading.md#polyfills
-$polyfillsBootstraps = \array_map(
-    static fn (SplFileInfo $fileInfo) => $fileInfo->getPathname(),
-    \iterator_to_array(
-        Finder::create()
-            ->files()
-            ->in(__DIR__ . '/vendor/symfony/polyfill-*')
-            ->name('bootstrap*.php'),
-        false,
-    ),
-);
-
-$polyfillsStubs = \array_map(
-    static fn (SplFileInfo $fileInfo) => $fileInfo->getPathname(),
-    \iterator_to_array(
-        Finder::create()
-            ->files()
-            ->in(__DIR__ . '/vendor/symfony/polyfill-*/Resources/stubs')
-            ->name('*.php'),
-        false,
-    ),
-);
```

Comment:
are polyfills automatically handled in the new Box? As we had a bug with them recently, unfortunately without a reproducer tests as we need to build PHP without some extensions (e.g. without `ext-intl`)

Reply:
It should now be handled yes; I'll double check however